databases/week2/school_database.sql

```diff
@@ -0,0 +1,75 @@
+CREATE DATABASE school_database;
+Use school_database;
+CREATE TABLE `class` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `begins` DATETIME NOT NULL,
+  `ends` DATETIME NOT NULL,
+  PRIMARY KEY (`id`)
+);
+CREATE TABLE `student` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+  `class_id` int(10) UNSIGNED NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`) ON DELETE CASCADE
+);
+insert INTO
```

Comment:
Uppercase (applies to the statements below as well)

Reply:
sure.. i just do how it comes with tab sometimes ...